frontend/app/src/main/res/layout/dialog_join_session.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center_horizontal"
+    android:orientation="vertical">
+
+    <EditText
+        android:id="@+id/session_code_edit_text"
```

Comment:
but i see you used _text for a non textview so maybe ill change mine then

Reply:
Let's stick with text for regular TextView and edit_text for EditText